src/main/view/game/TechOverviewPanel.java

```diff
@@ -2,28 +2,223 @@
 
 import java.awt.Point;
 
+import javafx.event.EventHandler;
+import javafx.scene.Group;
+import javafx.scene.canvas.Canvas;
 import javafx.scene.canvas.GraphicsContext;
+import javafx.scene.control.ScrollPane;
+import javafx.scene.control.ScrollPane.ScrollBarPolicy;
+import javafx.scene.input.MouseEvent;
+import javafx.scene.input.ScrollEvent;
+import javafx.scene.paint.Color;
 import view.GameModelAdapter;
 import view.ViewEnum;
 import view.assets.AssetManager;
 
 public class TechOverviewPanel extends OverviewPanel{
-
-	public TechOverviewPanel(GameModelAdapter gameModelAdapter, AssetManager assets, ViewEnum view) {
+	private static final int PANE_WIDTH = 219;
+	private static final int PANE_HEIGHT = 80;
+	private ScrollPane scrollPane = new ScrollPane();
+	private TechViewItem fertilizer = new TechViewItem(getAssets(), "Fertilizer", 
```

Comment:
I don't really know if it makes sense for the Panel class to do that, but I can change it.